decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -133,12 +135,7 @@ class Meeting < Meetings::ApplicationRecord
 
       scope :authored_by, ->(author) { where(decidim_author_id: author) }
 
-      scope_search_multi :with_any_type, TYPE_OF_MEETING.map(&:to_sym)
-
-      TYPE_OF_MEETING.each do |type|
-        scope type.to_sym, -> { where(type_of_meeting: type.to_sym) }
-        scope "not_#{type}".to_sym, -> { where.not(type_of_meeting: type.to_sym) }
-      end
+      scope_search_multi :with_any_type, TYPE_OF_MEETING.keys.map(&:to_sym)
```

Comment:
Isn't it already a symbol? I think we can remove `.to_sym`.
```suggestion
      scope_search_multi :with_any_type, TYPE_OF_MEETING.keys
```